src/execute.ts

```diff
@@ -13,23 +13,17 @@ import {settings} from './settings'
  * @example For ESM dev.js
  * ```
  * #!/usr/bin/env -S node --loader ts-node/esm --no-warnings=ExperimentalWarning
- * async function main() {
- *   const oclif = await import('@oclif/core')
- *   await oclif.execute({development: true, dir: import.meta.url})
- * }
+ * import { execute } from "@oclif/core"
  *
- * await main()
+ * await oclif.execute({development: true, dir: import.meta.url})
```

Comment:
With `import {execute} from '@oclif/core'`, there's no longer an oclif namespace - so it should now be `execute({development: true, dir: import.meta.url})`